translations/en.i18n.json

```diff
@@ -13,6 +13,7 @@
       "quiz_type": "Quiz",
       "live_type": "Live",
       "image_type": "Image",
+      "document_type": "Document",
```

Comment:
is there a i18n ticket ?

Reply:
yes https://kaltura.atlassian.net/browse/FEC-13644